packages/apollo-server/src/index.ts

```diff
@@ -31,6 +35,22 @@ export class ApolloServer<Context> extends ExpressServer {
 
     opts.app = express();
     super(opts);
+
+    opts.app.use('/.health-check', (req, res, next) => {
```

Comment:
I'd suggest we use `/.well-known/apollo/server-health` here as the default health check path, to mirror the functionality of Apollo Engine (and keep things off the root path).  It's also possible that should provide the ability to disable it completely.

Reply:
Good thought! I added a disable